test/test_librosa_compatibility.py

```diff
@@ -269,15 +269,14 @@ def test_basics4(self):
         }
         self.assert_compatibilities(**kwargs)
 
-    @unittest.skipIf("sox" not in common_utils.BACKENDS, "sox not available")
-    @common_utils.AudioBackendScope("sox")
     def test_MelScale(self):
         """MelScale transform is comparable to that of librosa"""
         n_fft = 2048
         n_mels = 256
         hop_length = n_fft // 4
 
         # Prepare spectrogram input. We use torchaudio to compute one.
+        common_utils.set_audio_backend('default')
```

Comment:
this test shouldn't pass on windows with default backend and mp3?

Reply:
see [test](https://app.circleci.com/pipelines/github/pytorch/audio/2226/workflows/7cd2cd8a-f060-4dce-a71f-d2c93b54fa32/jobs/57860/steps) failing